### ISSUE TITLE
Support Multiple Projects

### ISSUE THREAD

**Issue by fescobar** [fescobar/allure-docker-service#77]
This feature will allow you to handle multiple projects/reports to be used for an organization.

**Comment by fescobar** [comment]
I've released a `beta` version `frankescobar/allure-docker-service:beta`. Please if you have any other `beta` version remove your previous images and download the latest `beta`

I've implemented this feature requested. For this change, I needed to deprecate the port `4040`, just you need to use the port `5050` like this:

```
version: '3'
services:
  allure:
    image: "frankescobar/allure-docker-service:beta"
    environment:
      CHECK_RESULTS_EVERY_SECONDS: 1
      KEEP_HISTORY: "TRUE"
    ports:
      - "5050:5050"
    volumes:
      - ${PWD}/projects:/app/allure-docker-api/static/projects
```
Have in mind for multiple projects we are using this internal path: `/app/allure-docker-api/static/projects` that is different for a single project

Also, instead of using the url http://localhost:4040, you should use http://localhost:5050/latest-report.
https://github.com/fescobar/allure-docker-service/tree/beta#port-4040-deprecated

The documentation is up to date in the README.md from `beta` branch
https://github.com/fescobar/allure-docker-service/tree/beta

You can follow this section to see how it works `Multiple Projects`:
https://github.com/fescobar/allure-docker-service/tree/beta#MULTIPLE-PROJECTS---REMOTE-REPORTS

Also, you can check the way how you can navigate between different report executions. https://github.com/fescobar/allure-docker-service/tree/beta#keep-history-and-trends
Before releasing this, I will add a way to keep just a specific quantity of executions to avoid overload the storage in the machine.

I appreciate your suggestions and please let me know if everything is working well or not.

**Comment by karolxavierleite** [comment]
Nice improvement, I'm testing 👍

**Comment by fescobar** [comment]
Thank you @karolxavierleite. I've recently released a new `beta` version (remove your previous beta version if you have one locally) because I've found some issues wich I've already fixed.

**Comment by karolxavierleite** [comment]
LGTM! When you release it we will use for sure! It will be very helpful to us!! Thanks

**Comment by fescobar** [comment]
The tests were successful!!. I've re-released version `2.13.3`, please remove your local images if you are using the same version to pull the latest `2.13.3`.
Check the documentation about the new changes here:
https://github.com/fescobar/allure-docker-service#allure-docker-service-1
https://github.com/fescobar/allure-docker-service/releases/tag/v2.13.3
